packages/suite/src/views/settings/SettingsGeneral/Experimental.tsx

```diff
@@ -49,6 +51,10 @@ const FeatureLine = ({ feature, enabledFeatures }: FeatureLineProps) => {
             },
         });
     };
+    const handleClick = () => {
+        if (!config.routeName) return;
+        dispatch(goto(config.routeName as Route['name']));
```

Comment:
```suggestion
        dispatch(goto(config.routeName));
```
And instead change  `routeName?: Route['name']` in `experimental.ts` :slightly_smiling_face:

Reply:
Fixed in 1f2c2b5961e9da0af54d36f662b25619149744ba